### DISCUSSION TITLE
视频合集排序

### DISCUSSION THREAD

**Discussion by zhanshou** [the1812/Bilibili-Evolved#5065] [功能建议 / Ideas]
UP会为自己的视频建立一个合集来分类归纳,但是顺序非常乱,
B站只有默认排序和升序排序,但是都没有任何规律可言,
我问过客服,说这个排序完全就是UP自己排的确实没有规律
所以如果UP不对自己的合集内的视频排序,那观众看到的就是乱的

**Comment by Geeyun-JY3** [comment]
#2368、https://github.com/the1812/Bilibili-Evolved/discussions/1930#discussioncomment-849494

按理说这应该是破站的 bug（个人感觉个人空间有不少 bug 的，比如个人空间的搜索，有时候有些关键词根本搜不出想要的结果，或者命中的结果泛得要死），个人空间近期有大改：#5011，也不知道顺便修了没有，可惜我现在也进不去这个灰度去验证了。
